lib/transport/objects.ts

```diff
@@ -17,6 +17,7 @@ export interface TrackHeader {
 export interface TrackChunk {
 	group: number // The group sequence, as a number because 2^53 is enough.
 	object: number
+	status: number
 	payload: Uint8Array
```

Comment:
Status is mutually exclusive with payload so we can represent that I think. You'll just have to check if the type is Uint8Array.

```typescript
export interface TrackChunk {
	group: number // The group sequence, as a number because 2^53 is enough.
	object: number
	payload: Uint8Array | Status
}

export enum Status {
  OBJECT_NULL = 1,
  GROUP_NULL = 2,
  GROUP_END = 3,
  TRACK_END = 4,
}
```